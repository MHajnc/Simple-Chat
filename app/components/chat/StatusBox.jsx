import React from 'react';
import styles from './chat.css';

export default class StatusBox extends React.Component {
  render() {
    return (
      <div className={styles.chat__messageList__statusBox}>
        {this.props.status} - 
        there {this.props.count > 1 ? 'are' : 'is'} {this.props.count} {this.props.count > 1 ? 'participants' : 'participant'}
      </div>
    );
  }
}
