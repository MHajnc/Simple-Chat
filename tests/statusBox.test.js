import React from 'react';
import renderer from 'react-test-renderer';

import StatusBox from '../app/components/chat/StatusBox';

test('Testing StatusBox Component', () => {
  const out = renderer.create(
    <StatusBox status="Test on" count="123" />
  ).toJSON();

  expect(out).toMatchSnapshot();
});