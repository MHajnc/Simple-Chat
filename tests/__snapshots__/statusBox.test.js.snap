// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing StatusBox Component 1`] = `
<div
  className="chat__messageList__statusBox"
>
  Test on
   - there 
  are
   
  123
   participants
</div>
`;
